jogo42-cb/jogador.c

```diff
@@ -75,6 +75,10 @@ void MoverJog(GameState* gs)
     {
         gs->jog.pos = posFutura;
     }
+    // Ataque
+    gs->jog.hitbox = gs->jog.pos;
+    gs->jog.hitbox.x = gs->jog.hitbox.x + 60*cos(gs->jog.rot/60);
+    gs->jog.hitbox.y = gs->jog.hitbox.y + 60*sin(gs->jog.rot/60);
```

Comment:
Ver comentário da linha acima.
```suggestion
    gs->jog.hitbox.y = gs->jog.hitbox.y + RAD2DEG * sin(gs->jog.rot / DEG2RAD);
```

Reply:
Consertei com commit 9523dee